kitt_bar_app/current_batch.rb

```diff
@@ -40,14 +40,23 @@ def header
   end
 
   def tickets
-	  puts "- Tickets|href=#{tickets_url}|size=12"    
+	  puts "- Tickets|size=12"
+
+    @api_data.dig('tickets').each { |ticket|
+      puts "-- #{ticket.dig('user', 'name')} #{assigned_ticket(ticket)}"
+      ## TODO : two different puts for Take or Leave
+      # url = "https://kitt.lewagon.com/api/v1/tickets/#{ticket.dig('id')}/take"
+      puts "---- #{Color.orange}take it !#{Color.reset} | #{HttpKitt.put(ticket, "take")}" if ticket.dig('policy', 'current_user_can_take')
+      puts "---- #{Color.green}mark as done !#{Color.reset} | #{HttpKitt.put(ticket, "done")}" if ticket.dig('policy', 'current_user_can_mark_as_solved')
+      puts "---- #{Color.red}cancel#{Color.reset} | #{HttpKitt.put(ticket, "cancel")}" if ticket.dig('policy', 'current_user_can_cancel')
+    }
   end
 
   def end_ticket
     return unless @ticket
 
-    url = "https://kitt.lewagon.com/api/v1/tickets/#{@ticket['id']}/mark_as_solved"
-    puts "- Validate ticket|shell=\"#{__dir__}/ticket_validator.rb\" param1=#{KITT_COOKIE} param2=#{url}"
+    # url = "https://kitt.lewagon.com/api/v1/tickets/#{@ticket['id']}/mark_as_solved"
+    puts "- Validate ticket with #{@ticket.dig('user','name')} | #{HttpKitt.put(@ticket, "done")}"
```

Comment:
Is this still relevant as it seems to be implemented in the `tickets` method above?

Reply:
I think it's nice to still have a quick "validate ticket" button without having to go down in the tickets menu > name > validate

Reply:
Indeed 🤓